apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_utils.py

```diff
@@ -197,8 +210,13 @@ def from_pretrained(
                 use_base_vae=use_base_vae,
                 use_tuned=use_tuned,
             )
-            clip, unet, vae = mlir_import()
-            return cls(vae, clip, get_tokenizer(), unet, scheduler)
+            clip, unet, vae_encode, vae = mlir_import()
+            return cls(vae_encode, vae, clip, get_tokenizer(), unet, scheduler)
         return cls(
-            get_vae(), get_clip(), get_tokenizer(), get_unet(), scheduler
+            get_vae_encode(),
+            get_vae(),
+            get_clip(),
+            get_tokenizer(),
+            get_unet(),
+            scheduler,
```

Comment:
This will break Text2Image. Instead create a from_pretrained method inside the Inpaint class itself as the vae_encode is specific to the Inpainting.